cmd/apps.go

```diff
@@ -85,8 +86,14 @@ func AppsList(results int) error {
 
 	fmt.Printf("=== Apps%s", limitCount(len(apps), count))
 
+	var appNames []string
```

Comment:
I will update my pull request, but first at https://github.com/deis/deis/pull/5072

Do you want seperate commits for each action or can I just squash everything together at the end?